src/lucky/renderable.cr

```diff
@@ -19,13 +19,14 @@ module Lucky::Renderable
         {{ key }}: {{ key }},
       {% end %}
     )
-    log_html_render(context, view)
+    debug_message = log_message(view)
     body = view.perform_render.to_s
-    Lucky::Response.new(context, "text/html", body)
+    Lucky::Response.new(context, "text/html", body,
+                        debug_message: debug_message)
```

Comment:
I think you could remove the intermediate variable and call `log_message(view)` here directly.

Also can you run `crystal tool format` to make sure this is formatted? You may need to put "text/html" on a newline for the formatter to put them all on newlines. I'm not sure.

Reply:
I'm always down to remove temp variables.

Reply:
Of note, the formatter doesn't seem to have an opinion on how to indent the arguments here.